resources/istio-kyma-patch/templates/clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
```

Comment:
I would go with Role in ns `istio-system` rather than ClusterRole

Reply:
ClusterRole is needed, because patch needs access to crds, which are cluster resource.

Reply:
ClusterRole was split in two, as suggested below.